contributed_definitions/nyaml/NXpeak_model.yaml

```diff
@@ -0,0 +1,140 @@
+category: base
+doc: |
+  Peak fitting of a one-dimensional spectrum.
+symbols:
+  doc: |
+    The symbols used in the schema to specify e.g. dimensions of arrays.
+  n_peaks: |
+    Number of peaks.
+type: group
+NXpeak_model(NXprocess):
+  region_REGION(NXfit_region):
+    doc: |
+      Subset of the abscisssa that is used for fitting.
+      It is envisioned that regions are labeled as regions_0, regions_1, and so on.
+  peak_PEAK(NXpeak):
+    doc: |
+      One spectral peak of the peak model.
+      It is envisioned that peaks are labeled as peak_0, peak_1, and so on.
+  background(NX_background):
+    doc: |
+      Description of the function used for background subtraction.
+  envelope(NX_NUMBER):
+    unit: NX_ANY
+    doc: |
+      Resulting envelope of all NXpeak instances.
+      
+      Should have the same dimension as the `position` axes of the individual peaks.
```

Comment:
Define symbol for fit region number of points, and use here.

Reply:
done